conan/internal/util/runners.py

```diff
@@ -79,7 +79,7 @@ def detect_runner(command):
     return proc.returncode, "".join(output_buffer)
 
 
-def check_output_runner(cmd, stderr=None, ignore_error=False):
+def check_output_runner(cmd, stderr=None, ignore_error=False, ignore_encoding_error=False):
```

Comment:
Have you tried using the ``conan_run()`` instead, which already has a ``ignore=True`` decoding policy as default?